packages/web-client/app/services/workflow-persistence.ts

```diff
@@ -0,0 +1,23 @@
+import { default as Service } from '@ember/service';
+
+export default class WorkflowPersistence extends Service {
```

Comment:
Perhaps we should consider test behavior for this that uses a mock localstorage?

Reply:
Yeah, sounds good. Will consider this when I start writing tests.